aws/resource_aws_security_group_rule_test.go

```diff
@@ -168,6 +170,18 @@ func TestAccAWSSecurityGroupRule_Ingress_Source_With_Account_Id(t *testing.T) {
 				Config: testAccAWSSecurityGroupRule_Ingress_Source_with_AccountId(rInt),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAWSSecurityGroupRuleExists("aws_security_group.web", &group),
+					resource.TestMatchResourceAttr(
```

Comment:
i don't think a change is needed but as a note, you can also compare to the `security_group` as an alternative to matching the regex by doing something like:
```
resource.TestCheckResourceAttrPair(ruleName, "security_group_id", "aws_security_group.web", "id")
```

Reply:
Good point! Updated!